From fbc78262d128156e699f3bf7923f31dc56c42226 Mon Sep 17 00:00:00 2001
From: Michal Schmidt <mschmidt@redhat.com>
Date: Fri, 5 Sep 2014 19:49:37 -0400
Subject: [ethernet] bnx2x: Convert return 0 to return rc

Message-id: <1409946615-5862-33-git-send-email-mschmidt@redhat.com>
Patchwork-id: 91605
O-Subject: [RHEL7.1 PATCH 32/70] bnx2x: Convert return 0 to return rc
Bugzilla: 1089402
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Nikolay Aleksandrov <nikolay@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1089402

Upstream commit:
commit 02948344fbf511c6eec28687dc76bb64b758ff93
Author: Joe Perches <joe@perches.com>
Date:   Thu May 15 23:51:43 2014 -0700

    bnx2x: Convert return 0 to return rc

    These "return 0;" uses seem wrong as there are
    rc variables where error return values are set
    but unused.

    Signed-off-by: Joe Perches <joe@perches.com>
    Acked-by: Dmitry Kravkov <Dmitry.Kravkov@qlogic.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/broadcom/bnx2x/bnx2x_sriov.c b/drivers/net/ethernet/broadcom/bnx2x/bnx2x_sriov.c
index 81cc2d9..b8078d5 100644
--- a/drivers/net/ethernet/broadcom/bnx2x/bnx2x_sriov.c
+++ b/drivers/net/ethernet/broadcom/bnx2x/bnx2x_sriov.c
@@ -2695,7 +2695,7 @@ out:
   bnx2x_unlock_vf_pf_channel(bp, vf, CHANNEL_TLV_PF_SET_MAC);
  }
 
- return 0;
+ return rc;
 }
 
 int bnx2x_set_vf_vlan(struct net_device *dev, int vfidx, u16 vlan, u8 qos)
diff --git a/drivers/net/ethernet/broadcom/bnx2x/bnx2x_vfpf.c b/drivers/net/ethernet/broadcom/bnx2x/bnx2x_vfpf.c
index 0c067e8..784c715 100644
--- a/drivers/net/ethernet/broadcom/bnx2x/bnx2x_vfpf.c
+++ b/drivers/net/ethernet/broadcom/bnx2x/bnx2x_vfpf.c
@@ -747,7 +747,7 @@ int bnx2x_vfpf_config_mac(struct bnx2x *bp, u8 *addr, u8 vf_qid, bool set)
 out:
  bnx2x_vfpf_finalize(bp, &req->first_tlv);
 
- return 0;
+ return rc;
 }
 
 /* request pf to config rss table for vf queues*/
-- 
1.7.1